0000-template.md

```diff
@@ -0,0 +1,25 @@
+# Feature Name
+
+## Summary
+[summary]: #summary
+
+One paragraph explanation of the feature.
+
+## Motivation
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support?
+What is the expected outcome?
+
+## Approach
+[approach]: #approach
+
+Provide a brief description of the implementation details for the feature.
+What existing code paths are you planning to modify.
```

Comment:
End questions with a question mark. Same on the next line.

Reply:
Done